.ci/update_docs.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+source ../rultor_secrets.sh
+
+set -x
+set -e
```

Comment:
the sets are to prevent the source from showing the passwords, right? So they should be *before* the sourcing, right?